### PR TITLE
update all release channels

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: data/project/ember/lts.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.8.1
-initialReleaseDate: 2019-04-02
-lastRelease: 3.8.1
-futureVersion: 3.8.2
+initialVersion: 3.12.0
+initialReleaseDate: 2019-08-06
+lastRelease: 3.12.1
```
Comment (igorT) [inline data/project/ember/lts.md]: This should be 3.12.4

File: data/project/ember/lts.md [hunk 2]
```diff
@@ -5,12 +5,12 @@ filter:
...
+futureVersion: 3.12.2
```
Comment (igorT) [inline data/project/ember/lts.md]: 3.12.5

File: data/project/ember/beta.md [hunk 3]
```diff
@@ -5,14 +5,14 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.13.0
-lastRelease: 3.14.0-beta.2
-futureVersion: 3.14.0
-finalVersion: 3.14.0
+initialVersion: 3.15.0
+lastRelease: 3.15.0-beta.3
```
Comment (igorT) [inline data/project/ember/beta.md]: 3.15.0-beta.0

**Comment by kategengler** [comment]
I thought we had gotten rid of the timeline, which is why I have not been updating. @jenweber?

**Pull request merged**